Monika After Story/game/script-topics.rpy

```diff
@@ -13016,6 +13016,43 @@ label monika_unknown:
+    m 3eua "While the genre's popularity has surged only recently, the first known example of alternate history speculation dates back to the times of Ancient Rome..."
+    m 3eud "A Roman historian named Livy contemplated a scenario in his history book, '{i}Ab Urbe Condita Libri{/i}', where Alexander the Great had surived his fatal fever."
+    m 2eud "A hypothetical scenario would have him pursue his plans of European expansion and inevitable confrontation of the Argead Empire with the Romans..."
+    m 2euc "Livy ultimately concluded that this scenario would likely end up becoming a Roman victory."
+    m 3eua "One of the better known contemporary examples is Robert Harris' detective novel '{i}Fatherland{/i}' which depicts a world where the Third Reich won the Second World War."
+    m 1eud "The book is set in alternate 1964 and follows the story of a Kripo detective who is investigating the murder of a high-ranking Nazi Party official..."
+    m 3euc "As he uncovers more details, he realizes that he is caught up in a political scandal involving senior party officials who are apparently being murdered under staged circumstances."
+    m 1eud "He ultimately discovers the truth behind the murders and the culprit is revealed to be Reinhard Heydrich, now leader of the SS. And the kicker? He's supposed to be long dead."
+    m 1euc "In our timeline, Heydrich was fatally wounded by Allied operatives in Prague in 1942. This is an earliest point of divergence from the original timeline in the story."
+    m 3esd "The reveal of Heydrich being well and alive in that timeline also indicates that his survival somehow influenced the outcome of the war in the favor of Nazi Germany."
+    m 2ekd "Anyway, the murders were ordered as a ploy to cover up the Holocaust, a fact that mostly remained secret to German public and discredited as a conspiracy theory..."
+    m 2dkc "And since the murderer is the leader of the Nazi state sec, it is him who ends up becoming the one pursuing and arresting the detective, not the other way around."
+    m 2rkc "Therefore, the novel itself subverts the expectations of the reader who expects it to be another typical crime story, albeit set in a dystopian scenario."
+    m 2esc "As you can see, one divergence can drastically alter the outcome of the entire timeline, as it's bound to create a domino effect."
+    m 4eud "And this ultimately leads to the butterfly effect, which negates the existence of many other events, as the circumstances behind them may simply no longer exist."
+    m 1eud "That makes me wonder... if some major events went differently in your world, would my creator still come up with the idea for this game?"
+    m 1rksdld "Even so, its entire concept could be vastly different... perhaps I wouldn't be able to have my epiphany like I did or even not exist at all."
+    m 1dksdlc "That means we wouldn't be able to meet for real, either. The thought itself chills me to the core..."
```

Comment:
I don't think it's too far off, but I'm not too sure Monika would say the expression "to the core".